### PR TITLE
Ranger general, story fix, historical ranger-generals for game.

### PR DESCRIPTION
**Pull request by aybarsavci**
## Changelog:
- Sylvanas is the Ranger-General of Quel'thalas in the year 603, her mother, Lireesa in the 583 year and Halduron in the 605 year.

# How to test:
Just play as Quel'thalas and look at marshals (ranger-generals).

### CODE REVIEW

**Comment by zumbak04** [comment]
Fixed the changelog, take a look.

File: history/characters/42000_high_elf.txt [hunk 1]
```diff
@@ -1254,8 +1254,15 @@
 		}
 	}
 	620.4.5={ death=yes }
+
+	603.1.1 = {
+		effect = {
+            give_job_title = job_marshal    
+        	}
+	}
```
Comment (zumbak04) [inline history/characters/42000_high_elf.txt:1262]: It's not a syntax error, but better keep dates in chronological order. Thus the 620 year goes after 603, not vice versa.

Comment (zumbak04) [inline history/characters/42000_high_elf.txt:1262]: To make sure the position is free.
```suggestion
	603.1.1 = {
		effect = {
			liege = { job_marshal = { leave_council_position_no_opinion_effect = yes } }
			leave_council_position_no_opinion_effect = yes
			give_job_title = job_marshal
        	}
	}
```

File: history/characters/42000_high_elf.txt [hunk 2]
```diff
@@ -1277,6 +1284,11 @@
 	}
 	620.2.9={ death=yes }
 }
+	605.6.6 = {
+		effect = {
+            give_job_title = job_marshal    
+        }
+	}
```
Comment (zumbak04) [inline history/characters/42000_high_elf.txt:1291]: To make sure the position is free.
```suggestion
	605.6.6 = {
		effect = {
			liege = { job_marshal = { leave_council_position_no_opinion_effect = yes } }
			leave_council_position_no_opinion_effect = yes
			give_job_title = job_marshal
        	}
	}
```

File: history/characters/42000_high_elf.txt [hunk 3]
```diff
@@ -2363,6 +2375,11 @@
 		}
 		death = { death_reason = death_battle }
 	} 
+	583.1.1 = {
+		effect = {
+            give_job_title = job_marshal    
+        }
+	}
```
Comment (zumbak04) [inline history/characters/42000_high_elf.txt:2382]: To make sure the position is free.
```suggestion
	583.1.1 = {
		effect = {
			liege = { job_marshal = { leave_council_position_no_opinion_effect = yes } }
			leave_council_position_no_opinion_effect = yes
			give_job_title = job_marshal
        	}
	}
```

**Comment by aybarsavci** [comment]
I think its done perfectly.

Last Update:
#Edited Position of History 
#Made Sure Position is Free

File: history/characters/42000_high_elf.txt [hunk 4]
```diff
@@ -1214,6 +1214,13 @@
 	550.1.1={
 		employer=42002 #Anasterian
 	}
+	603.1.1 = {
+		effect = {
+			liege = { job_marshal = { leave_council_position_no_opinion_effect = yes }
```
Comment (zumbak04) [inline history/characters/42000_high_elf.txt:1219]: `}` is missing.
```suggestion
			liege = { job_marshal = { leave_council_position_no_opinion_effect = yes } }
```

File: history/characters/42000_high_elf.txt [hunk 5]
```diff
@@ -1275,8 +1285,16 @@
 		add_trait=creature_blood_elf
 		culture=blood_elf
 	}
+	605.6.6 = {
+		effect = {
+			liege = { job_marshal = { leave_council_position_no_opinion_effect = yes }
```
Comment (zumbak04) [inline history/characters/42000_high_elf.txt:1290]: `}` is missing.
```suggestion
			liege = { job_marshal = { leave_council_position_no_opinion_effect = yes } }
```

File: history/characters/42000_high_elf.txt [hunk 6]
```diff
@@ -2357,12 +2375,20 @@
 		employer=42002 #Anasterian
 		add_trait = class_hunter_7
 	}
+	583.1.1 = {
+		effect = {
+			liege = { job_marshal = { leave_council_position_no_opinion_effect = yes }
```
Comment (zumbak04) [inline history/characters/42000_high_elf.txt:2380]: `}` is missing.
```suggestion
			liege = { job_marshal = { leave_council_position_no_opinion_effect = yes } }
```

File: history/characters/42000_high_elf.txt (hunk 4 above)
Comment (arithon) [reply history/characters/42000_high_elf.txt:1219]: done

File: history/characters/42000_high_elf.txt (hunk 5 above)
Comment (arithon) [reply history/characters/42000_high_elf.txt:1290]: done

File: history/characters/42000_high_elf.txt (hunk 6 above)
Comment (arithon) [reply history/characters/42000_high_elf.txt:2380]: done

**Review by zumbak04 — APPROVED** [review]

**Pull request merged**